### DISCUSSION TITLE
Few of the Devanaagari (Unicode) characters not correctly displaying in plain editor

### DISCUSSION THREAD

**Discussion by jiggneshhgohel** [standardnotes/forum#2756] [General]
Hello,

I am writing some content involving words in Sanskrit language in the plain editor however some of them are not displaying correctly.

**Expected behavior**

The words as specified below should be displayed however some of the words are not correctly displaying as shown in the screenshot 

```
ॐ नमस्ते गणपतये | त्वमेव प्रत्यक्षं तत्वमसि | त्वमेव केवलं कर्तासि | त्वमेव केवलं धर्तासि | त्वमेव केवलं हर्तासि | त्वमेव खल्विदं ब्रह्मासि | त्वं साक्षादात्मासि नित्यम् ||
```

**Screenshots**

Note how the words प्रत्यक्षं , ब्रह्मासि are not seen displayed correctly in the screenshot below taken from Desktop app.

![Screenshot from 2021-09-08 13-18-51](https://user-images.githubusercontent.com/10440841/132468518-9652d8e6-269e-4fc1-bac6-61cc13928db5.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04

The issue above is only on Desktop-app on Ubuntu 18.04 machine. On Mobile app the characters are displaying correctly. 

The Desktop-app version I am using is 3.4.2 and on Mobile app (in my iPhoneSE) I always update whenever updates are available.**

Am I missing any setting or this is something which should be fixed in the application?

Thanks.

**Comment by JaspalSuri** [comment]
Hi @jiggneshhgohel, I'm incredibly sorry for delayed response. I'm sorry to see that the characters aren't rendering correctly. 

While I haven't been able to reproduce the issue on our [web](https://app.standardnotes.com) and desktop apps, I've noticed that you're using an older version of the desktop app. I'm sorry that it wasn't automatically updated to the latest version, v3.8.21; please grab the latest version from our [downloads page](https://standardnotes.com/download). If even after updating to the latest version doesn't work, please check the web app (you don't have to sign in, simply create a new note and delete it if needed), and then check the Software Updater app to see if there are any language-related packages that need to be updated. I am using a newer LTS of Ubuntu, version 20.04, so although unlikely, it's possible that the issue pertains to your version of Ubuntu. 

Another issue might be that it isn't working in the app that is provided by the store, which is a Snap, but may only work in the AppImage. If you're using the Snap but are a bit apprehensive of using the AppImage, you can use a 3rd party cool called AppImageLauncher to simplify the installation and update processes.

**Comment by jiggneshhgohel** [comment]
@JaspalSuri Thanks for your response.

I updated the Desktop App to latest version 3.8.21 but still the characters, I reported before, are not rendering correctly. I tried your following suggestion

>If even after updating to the latest version doesn't work, please check the web app (you don't have to sign in, simply create a new note and delete it if needed), and then check the Software Updater app to see if there are any language-related packages that need to be updated.

and on web-version the characters are rendering correctly. Please see the screenshot below

![Screenshot from 2021-09-13 12-58-29](https://user-images.githubusercontent.com/10440841/133041913-a5d20a51-49f1-4a92-bb96-9060612c849b.png)

Regarding following

> Another issue might be that it isn't working in the app that is provided by the store, which is a Snap, but may only work in the AppImage. If you're using the Snap but are a bit apprehensive of using the AppImage, you can use a 3rd party cool called AppImageLauncher to simplify the installation and update processes.

From the start itself (as in when I found StandardNotes and started using it) I am using AppImage version.

Regarding upgrading Ubuntu to latest LTS version 20.04 I am sorry I cannot attempt the OS upgrade because of some personal reasons.

Also in browser and in an other terminal app Konsole the characters are rendering correctly so I don't think this problem I have reported has its root cause in some missing language pack or configuration in Operating System.

Hope above information helps in trying to figure out the root cause. 

Thanks for all your help. Really appreciate it.
